functions/triangle.js

```diff
@@ -31,7 +52,17 @@
  * @param {number} c
  * @return {boolean} whether the triangle is scalene
  */
-// YOUR CODE HERE
+
+ function isScalene(a, b, c){
```

Comment:
Check out this function again - but aside from that, well done with the other functions!